src/components/Identification/Components/LinksDialog/index.js

```diff
@@ -10,8 +10,8 @@ import {
   Typography,
   TextField,
 } from '@material-ui/core';
-import { socialMediaInfo } from '../../../../../../socialMedia';
-import { gordonColors } from '../../../../../../theme';
+import { socialMediaInfo } from '../../../../socialMedia';
+import { gordonColors } from '../../../../theme';
 import './linksDialog.css';
```

Comment:
I moved the Identification component from inside the My Profile component to "src/components". Due to this, I had to change the file paths of imported files